.github/ISSUE_TEMPLATE/new-ontology.yml

```diff
@@ -0,0 +1,249 @@
+name: Request a New Ontology
+description: Request to register a new ontology with the OBO Foundry.
+title: Request for new ontology [NAME]
+labels: [ new ontology ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here: http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html
+  - type: input
+    id: title
+    attributes:
+      label: Title
+      description: What is the full title of your ontology? Please do not include redundant acronyms.
+      placeholder: ex. Chemical Entities of Biological Interest
```

Comment:
I have never seen `ex:` used like that, maybe `e.g.:` is more readily understood (unsure)?

Reply:
Fixed in [706f12a](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1824/commits/706f12a9b1b82c52f3e4eba17dbf76d1cf1822bd)